spec/src/main/asciidoc/core/Introduction.adoc

```diff
@@ -23,35 +23,53 @@ application programmer._
 [[a6]]
 === What is New in This Release
 
-* _The 'javax' namespace/packages have been renamed to 'jakarta'_
+The Enterprise Beans 4.0 specification is a breaking change from
+prior Enterprise Beans specifications delivering a namespace change
+from `javax` to `jakarta` across the API along with removal of
+deprecated APIs.
 
-* _Removed the methods relying on java.security.Identity_
+* _All 'javax.ejb' namespace/packages have been renamed to 'jakarta.ejb'_
 
-* _Removed the methods relying on JAX-RPC_
+* _Removed the methods relying on java.security.Identity which has
+been removed from the Java 14._
 
-* _Removed the deprecated EJBContext.getEnvironment() method_
+* _Removed the methods relying on JAX-RPC to reflect the removal of
+JAX-RPC from the Jakarta EE 9 Platform._
 
-* _Removed the Support for Distributed Interoperability_
+* _Removed the deprecated EJBContext.getEnvironment() method._
+
+* _Removed the Support for Distributed Interoperability to reflect
+the removal of CORBA from Java 11 and the Jakarta EE 9 Platform._
 
 * _Marked the EJB 2.x API Group as "Optional"_
 
```

Comment:
In the current [spec document](https://github.com/eclipse-ee4j/ejb-api/blob/master/spec/src/main/asciidoc/core/RuntimeEnvironment.adoc), the term "Enterprise Beans 2.x API" is used. I think we should use the new term name.

Reply:
Note that isn't a change part of this PR.  That said, the trademark rules here are a little nuanced.  Here's the discussions we've had in the Specification/Steering committees and the guidance we've been given on use of either trademark.

 - When referring to the specification in general or referring to a new version of the specification, we should use the new name.
 - When explicitly referring to past JCP versions of the specification we should take care to use the appropriate trademark.  For example,  "J2EE 1.2" not "Jakarta EE 1.2"

Similarly, when "Java EE" became the new name for "J2EE" we continued to refer to the older specs using their proper name "J2EE 1.3", "J2EE 1.4", etc and did not being referring to them as "Java EE 1.3",  "Java EE 1.4", etc.

Reply:
+1 to change EJB 2.x to Enterprise Beans 2.x.

As I have explained in the mailing list, an earlier version under the new namespace should be referred with the new name.

Reply:
+1 to change EJB 2.x to Enterprise Beans 2.x.

This does not appear to be a reference to an older spec level, but instead is a reference to a subset of the current specification, specifically the 2.x API Group, so should use the same terminology as in the rest of the current version of the spec.